rush.json

```diff
@@ -592,6 +598,16 @@
         "@microsoft/rush-stack-compiler-3.2"
       ]
     },
+    {
+      "packageName": "@microsoft/rush-stack-compiler-3.3",
+      "projectFolder": "stack/rush-stack-compiler-3.3",
+      "reviewCategory": "libraries",
+      "shouldPublish": true,
```

Comment:
We can't merge this until the new package is created.

Reply:
Yup. I'm just going to keep this PR open until the package is created.